### PR TITLE
Issue 868: Mobile App feature: Add Successor in the case of death to profile page

### PR DESCRIPTION
**Pull request by KarinBerg**
See issue description for acceptance criteria: https://github.com/socialincome-san/public/issues/868

You can use the following Staging test user:
Number: +232 11 11 22 33
Username: Raphael Schoen

Screenshot_20240814-223702
Screenshot_20240814-223735

### CODE REVIEW

File: recipients_app/README.md [hunk 1]
```diff
@@ -31,7 +32,15 @@ setup.
 
 ## Rebuilding JSON Serialization
 
-flutter pub run build_runner watch --delete-conflicting-outputs
```
Comment (KarinBerg) [inline recipients_app/README.md:34]: The terminal told me that "flutter pub run" is deprecated and you should use "dart run" instead.

File: recipients_app/lib/data/models/phone.g.dart
Comment (KarinBerg) [inline recipients_app/lib/data/models/phone.g.dart:1]: phone.g.dart was changed after I ran "flutter pub run build_runner build --delete-conflicting-outputs"

File: recipients_app/lib/data/models/payment/social_income_payment.g.dart
Comment (KarinBerg) [inline recipients_app/lib/data/models/payment/social_income_payment.g.dart:1]: social_income_payment.g.dart was changed after I ran "flutter pub run build_runner build --delete-conflicting-outputs"

File: recipients_app/lib/l10n/app_en.arb [hunk 2]
```diff
@@ -60,7 +64,6 @@
     "call": "Call",
     "supportTeam": "Support Team",
     "phone": "Phone",
-    "email": "Email",
```
Comment (KarinBerg) [inline recipients_app/lib/l10n/app_en.arb:63]: Entry "email": "Email" was available twice

File: recipients_app/lib/l10n/app_kri.arb [hunk 3]
```diff
@@ -61,7 +64,6 @@
     "call": "Kɔl",
     "supportTeam": "Sɔpɔt Tim",
     "phone": "Fon",
-    "email": "Imel",
```
Comment (KarinBerg) [inline recipients_app/lib/l10n/app_kri.arb:64]: Entry "email": "Imel" was available twice

File: recipients_app/lib/l10n/app_en.arb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {
+    "@@locale": "en",
```
Comment (KarinBerg) [inline recipients_app/lib/l10n/app_en.arb:2]: linter fix

File: recipients_app/lib/view/pages/account_page.dart [hunk 5]
```diff
@@ -301,7 +305,9 @@ class AccountPageState extends State<AccountPage> {
                     final emailRegex = RegExp(
                       r"^[\w-]+(\.[\w-]+)*@[a-zA-Z0-9-]+(\.[a-zA-Z0-9-]+)*(\.[a-zA-Z]{2,})$",
                     );
-                    if (!emailRegex.hasMatch(value)) return localizations.errorEmailInvalid;
+                    if (!emailRegex.hasMatch(value)) {
+                      return localizations.errorEmailInvalid;
+                    }
```
Comment (KarinBerg) [inline recipients_app/lib/view/pages/account_page.dart:310]: linter fix

File: recipients_app/.vscode/settings.json [hunk 6]
```diff
@@ -1,3 +1,9 @@
 {
-	"java.configuration.updateBuildConfiguration": "automatic"
+	"java.configuration.updateBuildConfiguration": "automatic",
+	"dart.lineLength": 120,
+	"[dart]": {
+    "editor.rulers": [
+        120
+    ],
+}
```
Comment (KarinBerg) [inline recipients_app/.vscode/settings.json:8]: Set max line length for dart files to 120 to ensure all developers formatting with this rule. This avoid unnecessary changes in pull requests.

**Review by ssandino — APPROVED** [review]

**Review by MDikkii — APPROVED** [review]
LGTM 👏
